demos/yelp_demo/README.md

```diff
@@ -119,13 +95,13 @@ kubectl config use gke
 Create an environment:
 
 ```
-ks env add gke
+ks env add gke --namespace=kubeflow
 ```
 
 Install kubeflow on the cluster:
 
 ```
-ks apply gke -c kubeflow-core
+ks apply gke -c ambassador -c centraldashboard -c jupyterhub -c spartakus -c tf-job-operator
```

Comment:
why do separate components instead of core?

Reply:
`kfctl` creates them as individual components, so running `ks apply gke -c kubeflow-core` is a no-op